_posts/2020-04-17-snowplow-ios-tracker-1.3.0-released/index.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: "Snowplow iOS Tracker 1.3.0 released"
+title-short: Snowplow iOS Tracker 1.3.0
+tags: [snowplow, objc, ios, tracker]
+author: Alex Benini
+category: Releases
+permalink: /blog/2020/04/17/snowplow-ios-tracker-1.3.0-released/
```

Comment:
I set 24th because it seems improbable to publish this post today.

```suggestion
permalink: /blog/2020/04/24/snowplow-ios-tracker-1.3.0-released/
```